scripts/pre_commit_linter.py

```diff
@@ -1183,6 +1183,15 @@ def _check_docstrings(all_files):
                         is_docstring = True
                         is_class_or_function = False
 
+                # Check for space after """ in docstring.
+                if re.match(r'^""".+$', line) and is_docstring and (
+                        line[3] == ' '):
```

Comment:
Just confirming: does this check work for multiple spaces after `"""`?

Reply:
Yes it does. I tested it. Thanks!